src/main/resources/language/ru.yml

```diff
@@ -1,6 +1,6 @@
 # Russian(ru)
-# For BannerMaker v1.4.6
-# Contributor(s): sdir01
+# For BannerMaker v2.3.2
+# Contributor(s): maz1lovo
```

Comment:
Please do not remove other contributor(s) from list. 😅
```suggestion
# Contributor(s): sdir01, maz1lovo
```

Reply:
Sorry 😅